lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -308,6 +308,82 @@ def test_index_no_visible_section_in_chapter(self):
         self.assertNotIn('Problem 1', response.content)
         self.assertNotIn('Problem 2', response.content)
 
+    def _create_global_staff_user_and_course(self):
+        """
+        User will have staff access and creates the course
+        """
+        self.global_staff = GlobalStaffFactory.create()  # pylint: disable=W0201
+        self.client.login(username=self.global_staff.username, password='test')
+        course = CourseFactory.create()
```

Comment:
No, I think it's not a bug. but its `display_name` creates issues. that's why created the new one.